doc/options.yaml

```diff
@@ -1119,6 +1121,15 @@ adjointSolver:
   BCGS: BiCGStab
   IBCGS: Improved BiCGStab
 
+GMRESOrthogonalizationType:
+  desc: >
+    Which orthogonalization method to use with GMRES.
+    This will have the most effect on the adjoint solution, but the same method is also used in the ANK and NK solvers.
```

Comment:
Can you rephrase it slightly? The first time I read it I thought you were talking about the option's effect on the final adjoint state rather than the linear solver algorithm. The adjoint solution is the final linear state, adjoint solution process is the actual process, or you can call it the "adjoint solver". 

It is also true that it does affect the final solution. So maybe something like:
" This will affect the adjoint solver the most, but the same option is also used in the linear solvers in the ANK and NK solvers." in the main description, and then you can clarify a bit more about what you mean by "accuracy" in the explanation of each option below.